lbz/resource.py

```diff
@@ -207,6 +207,8 @@ def get_pagination(self, total_items: int, limit: int, offset: int) -> dict:
 
     @property
     def _pagination_uri(self) -> str:
-        if query_params := copy_without_keys(self.request.query_params, "offset", "limit"):
-            return f"{self.urn}?{urlencode(query_params)}&offset={{offset}}&limit={{limit}}"
+        if query_params := self.request.query_params.original_items("offset", "limit"):
```

Comment:
When I see it right now, I need to say that it is misleading 😬  I think, much more understandable would be to use keyword argument here:
```python
def original_items(self, keys_to_skip: Iterable[Hashable] = None) -> List[tuple]:
    keys_to_skip = keys_to_skip or []
    ...

self.request.query_params.original_items(keys_to_skip=["offset", "limit"])
```